### ISSUE TITLE
Dask Bag.distinct does not work with a key value

### ISSUE THREAD

**Issue by parkerhancock** [dask/dask#4777]
The Dask documentation indicates that [Bag.distinct](https://docs.dask.org/en/latest/bag-api.html#dask.bag.Bag.distinct) takes a keyword argument "key" that can be a callable or a string. However, the actual function throws an error when you attempt to give it a key. 

Example:

~~~python
import dask.bag as db
db.from_sequence(range(10)).distinct(key=lambda x: x)
~~~
Throws:
~~~
TypeError                                 Traceback (most recent call last)
<ipython-input-20-49ba2f85db9c> in <module>
----> 1 db.from_sequence(range(10)).distinct(key=lambda x: x)

TypeError: distinct() got an unexpected keyword argument 'key'
~~~

The helpstring for the function also does not show that distinct takes a key:
~~~
Help on method distinct in module dask.bag.core:

distinct() method of dask.bag.core.Bag instance
    Distinct elements of collection
    
    Unordered without repeats.
    
    >>> b = from_sequence(['Alice', 'Bob', 'Alice'])
    >>> sorted(b.distinct())
    ['Alice', 'Bob']
~~~

Maybe the documentation is ahead of the implementation?

Version info:
OS: OSX Mojave 10.14.4
Python: 3.6.7
Dask: 1.2.1

Thanks!

**Comment by parkerhancock** [comment]
Nvm. Went to go see what a fix would look like and found #4423

Should be fixed in the next release. 

It appears that the documentation is ahead of the release...

**Comment by parkerhancock** [comment]
Nvm. Went to go see what a fix would look like and found #4423

Should be fixed in the next release.

It appears that the documentation is ahead of the release...

**Comment by martindurant** [comment]
This was added in the last week, and not yet released.
